### PR TITLE
Convert gradle-mvn-mpp-push.gradle to KTS

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

**Comment by swankjesse** [comment]
Test failure is unrelated and I’m trying to understand more.
https://github.com/square/okio/pull/945

File: buildSrc/build.gradle.kts [hunk 1]
```diff
@@ -1,9 +1,12 @@
-import org.gradle.kotlin.dsl.`kotlin-dsl`
```
Comment (oldergod) [inline buildSrc/build.gradle.kts:1]: I _think_ you need this, don't you?

File: okio-fakefilesystem/build.gradle.kts [hunk 2]
```diff
@@ -80,4 +80,4 @@ dependencies {
   signature(deps.animalSniffer.javaSignature)
 }
 
-apply(from = "$rootDir/gradle/gradle-mvn-mpp-push.gradle")
+apply(plugin = "okio-publishing")
```
Comment (oldergod) [inline okio-fakefilesystem/build.gradle.kts:83]: I had no idea you could declare a local plugin like that

**Review by oldergod — APPROVED** [review]

File: buildSrc/build.gradle.kts (hunk 1 above)
Comment (swankjesse) [reply buildSrc/build.gradle.kts:1]: CI says no!

**Pull request merged**